### PR TITLE
Rename SignalR Server Transports

### PR DESCRIPTION
**Pull request by mikaelm12**
Adding "Server" to the server side transports. Mainly for search-ability.

### CODE REVIEW

File: src/SignalR/common/Http.Connections/src/Internal/Transports/LongPollingServerTransport.cs [hunk 1]
```diff
@@ -11,17 +11,17 @@
...
         private readonly ILogger _logger;
         private readonly CancellationToken _timeoutToken;
 
-        public LongPollingTransport(CancellationToken timeoutToken, PipeReader application, ILoggerFactory loggerFactory)
+        public LongPollingServerTransport(CancellationToken timeoutToken, PipeReader application, ILoggerFactory loggerFactory)
         {
             _timeoutToken = timeoutToken;
             _application = application;
-            _logger = loggerFactory.CreateLogger<LongPollingTransport>();
+            _logger = loggerFactory.CreateLogger<LongPollingServerTransport>();
```
Comment (BrennanConroy) [inline src/SignalR/common/Http.Connections/src/Internal/Transports/LongPollingServerTransport.cs]: If we're going to do this we should keep the old names for the loggers

**Comment by halter73** [comment]
Overall, I'm a big fan of this because as you say, it makes searching for the class easier. @BrennanConroy is right about the logging namespace change being breaking though.

**Review by halter73 — APPROVED** [review]
LGTM as long as we switch back to the old logging namespaces.

File: src/SignalR/common/Http.Connections/src/Internal/Transports/LongPollingServerTransport.cs [hunk 2]
```diff
@@ -11,17 +11,17 @@
...
         private readonly ILogger _logger;
         private readonly CancellationToken _timeoutToken;
 
-        public LongPollingTransport(CancellationToken timeoutToken, PipeReader application, ILoggerFactory loggerFactory)
+        public LongPollingServerTransport(CancellationToken timeoutToken, PipeReader application, ILoggerFactory loggerFactory)
         {
             _timeoutToken = timeoutToken;
             _application = application;
-            _logger = loggerFactory.CreateLogger<LongPollingTransport>();
+            _logger = loggerFactory.CreateLogger("LongPollingTransport");
```
Comment (BrennanConroy) [inline src/SignalR/common/Http.Connections/src/Internal/Transports/LongPollingServerTransport.cs]: I believe it's supposed to be "Microsoft.AspNetCore.Http.Connections.Internal.Transports.LongPollingTransport" ditto for the others

Comment (mikaelm12) [reply src/SignalR/common/Http.Connections/src/Internal/Transports/LongPollingServerTransport.cs]: Yeah, should be the full name

File: src/SignalR/common/Http.Connections/src/Internal/Transports/LongPollingServerTransport.cs [hunk 3]
```diff
@@ -21,7 +21,7 @@ public LongPollingServerTransport(CancellationToken timeoutToken, PipeReader app
         {
             _timeoutToken = timeoutToken;
             _application = application;
-            _logger = loggerFactory.CreateLogger("LongPollingTransport");
+            _logger = loggerFactory.CreateLogger("Microsoft.AspNetCore.Http.Connections.Internal.Transports.LongPollingTransport");
```
Comment (anurse) [inline src/SignalR/common/Http.Connections/src/Internal/Transports/LongPollingServerTransport.cs]: Comment as to why we're doing this.

File: src/SignalR/common/Http.Connections/src/Internal/Transports/ServerSentEventsServerTransport.cs [hunk 4]
```diff
@@ -21,7 +21,7 @@ public ServerSentEventsServerTransport(PipeReader application, string connection
         {
             _application = application;
             _connectionId = connectionId;
-            _logger = loggerFactory.CreateLogger("ServerSentEventsTransport");
+            _logger = loggerFactory.CreateLogger("Microsoft.AspNetCore.Http.Connections.Internal.Transports.ServerSentEventsTransport");
```
Comment (anurse) [inline src/SignalR/common/Http.Connections/src/Internal/Transports/ServerSentEventsServerTransport.cs]: Ditto, a comment as to why we're not using the `CreateLogger<T>` variant.

File: src/SignalR/common/Http.Connections/src/Internal/Transports/WebSocketsServerTransport.cs [hunk 5]
```diff
@@ -40,7 +40,7 @@ public WebSocketsServerTransport(WebSocketOptions options, IDuplexPipe applicati
             _options = options;
             _application = application;
             _connection = connection;
-            _logger = loggerFactory.CreateLogger("WebSocketsTransport");
+            _logger = loggerFactory.CreateLogger("Microsoft.AspNetCore.Http.Connections.Internal.Transports.WebSocketsTransport");
```
Comment (anurse) [inline src/SignalR/common/Http.Connections/src/Internal/Transports/WebSocketsServerTransport.cs]: Another comment!

**Comment by mikaelm12** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=271126&view=logs&jobId=64687d91-9896-58b2-003e-87130ea38281

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/2839

**Pull request merged**